src/setup/node.rs

```diff
@@ -60,7 +61,7 @@ impl Node {
         self
     }
 
-    pub async fn start(&mut self) -> Result<()> {
+    pub async fn start_process(&mut self) -> Result<()> {
```

Comment:
we probably want this one to be private, so that we only have a single function in the tests that creates and starts nodes